LayoutTests/fast/forms/range/sideways-lr-dragging.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <script src="../../../resources/js-test.js"></script>
+        <script>
+            function drag()
+            {
+                if (!window.testRunner)
+                    return;
+
+                const slider = document.getElementById("slider");
+
+                testRunner.dumpAsText();
+
+                shouldBe("slider.value", "'0'");
+               
+                debug("Dragging slider");
+                eventSender.mouseMoveTo(slider.offsetLeft + 7, slider.offsetTop + slider.offsetHeight - 7);
+                eventSender.mouseDown();
+                eventSender.mouseMoveTo(slider.offsetLeft + 7, slider.offsetTop + 7);
+                eventSender.mouseUp();
```

Comment:
Should be able to get this test working on iOS as well if you use `UIHelper.dragFromPointToPoint`.

Reply:
Done